src/legacy/core_plugins/kibana/public/dashboard/panel/panel_utils.ts

```diff
@@ -60,18 +72,33 @@ export class PanelUtils {
   // 1) decrease column height from 100 to 20.
   // 2) increase rows from 12 to 48
   // Need to scale pre 6.3 panels so they maintain the same layout
-  static convertPanelDataPre_6_3(panel, useMargins) { // eslint-disable-line camelcase
+  public static convertPanelDataPre_6_3(
+    panel: {
+      gridData: { w: number; x: number; h: number; y: number };
+      version: string;
+    },
+    useMargins: boolean
+  ) {
+    // eslint-disable-line camelcase
```

Comment:
Same as above, don't think this line is necessary here

Reply:
fc7f3db